internal/dataplane/translator/translator.go

```diff
@@ -266,6 +271,13 @@ func (t *Translator) InjectLicenseGetter(licenseGetter license.Getter) {
 	t.licenseGetter = licenseGetter
 }
 
+func (t *Translator) CustomEntityTypes() []string {
```

Comment:
Can KongClient's `Update()` be called asynchronously with Translator"s `BuildKongConfig()`? If so then there's a possible data race between line 276 and 222.

Reply:
No. The `BuildKongConfig()` method is only called in `Update()` (directly and in `tryRecoveringWithFallbackConfiguration` to build fallback configuration) so there is no data race risk here.